api/src/services/fields.ts

```diff
@@ -362,10 +362,10 @@ export class FieldsService {
 				: null;
 
 			if (
-				(hookAdjustedField.type === 'alias' ||
+				((hookAdjustedField.type === 'alias' && this.schema.collections[collection].fields[field.field]) ||
 					this.schema.collections[collection].fields[field.field]?.type === 'alias') &&
 				hookAdjustedField.type &&
-				hookAdjustedField.type !== this.schema.collections[collection].fields[field.field]?.type
+				hookAdjustedField.type !== this.schema.collections[collection].fields[field.field].type
```

Comment:
It seems that we are adding too many exceptions for this use case.
Do you know why do we need this if condition?
If this is related to the M2O↔️O2M recursive bug, maybe we can be explicit to it like check the relationship.
Also the error message is misleading since Dividers are Alias and can be changed 🤔

Reply:
Additionally, A possible separate issue causing this is due to the app sending the `type` field even though it's an unchanged field. 🤔

Reply:
To address the checking of relationship, that's another way to go around the issue but I feel that it would be better to be broader like current, by prevent type changes from / to alias types. The only addition now is to confirm if they exist in the schema. Similar to adding a TS optional chaining. 🤔 

Please make any changes accordingly, would love get this fix merged sooner. 👍 
Thanks for the review!